tests/e2e/support/objects/app-files/link/actions.ts

```diff
@@ -12,6 +12,13 @@ export interface createLinkArgs {
   space?: boolean
 }
 
+export interface copyLinkArgs {
+  page: Page
+  resource?: string
+  name?: string
+  via?: string
```

Comment:
Should `resource` be optional?
```suggestion
  resource: string
  name?: string
  via?: string
```